plugins/readme/src/components/ReadmeCard/ReadmeCard.tsx

```diff
@@ -39,7 +42,7 @@ export const ReadmeCard = ({ variant }: ReadmeCardProps) => {
         }
       >
         <div style={{ overflow: 'auto' }}>
-          <Box maxHeight={variant !== 'fullHeight' ? 235 : 500}>
+          <Box maxHeight={maxHeight}>
```

Comment:
i kinda agree with your points, here is my thoughts.

fullHeight property was added, but actually it was not full height, but only 500px height, that is not correct, as full height means what it means.

what i can do is:
1. by default max height is 235px
2. if variant == fullHeight max height would be set to none.

that means that i will remove maxHeight parameter as it will not be needed. thoughts?

Reply:
> Or, according to your issue, it should take all available height? This is not really solved in this PR since fullHeight is 235 px.

i kinda agree with your points, here is my thoughts.

fullHeight property was added, but actually it was not full height, but only 500px height, that is not correct, as full height means what it means.

what i can do is:

by default max height is 235px
if variant == fullHeight max height would be set to none.
that means that i will remove maxHeight parameter as it will not be needed. thoughts?

Reply:
I like that, let's go with that!